how-to/postgrest.md

```diff
@@ -397,4 +397,168 @@ Now, you can perform a keyword-based search directly from your web browser. Simp
+    <style>
+        .center-container {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            min-height: 100vh;
+        }
+    </style>
+  </head>
+  <body>
+    <div class="center-container">
+        <form name="search" id="searchForm">
+            <input type="search" size="60" maxlength="60" name="key" id="autoComplete">
+        </form>
+
+        <script src="https://cdn.jsdelivr.net/npm/@tarekraafat/autocomplete.js@10.2.7/dist/autoComplete.min.js">
+        </script>
+        <script type="text/javascript">
+            const searchForm = document.getElementById('searchForm');
+
+            searchForm.addEventListener('submit', async (event) => {
+                event.preventDefault();
+                const query = document.getElementById('autoComplete').value;
+                
+                if(query.length === 0) {
+                    return;
+                }
+
+                const response = await fetch(`http://localhost:3000/rpc/find_title?keywords=${query}`);
+                const result = await response.json();
+
+                const preElement = document.getElementById('output');
+                if(result.length > 0) {
+                    const formattedValue = JSON.stringify(result, null, 2);
+                    preElement.textContent = formattedValue;
+                } else {
+                    preElement.textContent = 'No results found';
+                }
+            });
+
+            const displayItems = (completions) => { };
+
+            const dataSource = async (query) => {
+                const source = await fetch(`http://localhost:3000/rpc/autocomplete?keyword=${query}`);
+                const data = await source.json();
+                const completions = data;
+                displayItems(completions);
+                return completions;
+            };
+
+            const autoCompleteJS = new autoComplete({
+                placeHolder: "Enter a keyword",
+                data: {
+                    src: dataSource
+                },
+                searchEngine: (query, record) => record,
+                events: {
+                    input: {
+                        selection: (event) => {
+                            const selection = event.detail.selection.value;
+                            autoCompleteJS.input.value = selection;
+                            dataSource(selection);
+                        }
+                    }
+                },
+                submit: true
+            });
+        </script>
+        <div id="result" style="margin-top: 7em;"></div>
+        <pre id="output"></pre>
+    </div>
+  </body>
+</html>
+```
+
+### Run PostgREST as API backend
+
+Run your PostgREST service using following command:
+
+```sh
+postgrest memo.conf
+```
+
+### Open html and Try Out
+
+Open `index.html` with your browser. 
+
+![PGroonga Auto Complete1](../images/postgres/pgautocomplete1.png)
```

Comment:
Could you fix a typo (`postgres`) in path?

```suggestion
![PGroonga Auto Complete1](../images/postgrest/pgautocomplete1.png)
```

Could you use `auto-complete1.png`, `auto-complete2.png` and so on naming?
(We don't need `pg` prefix because we only focus on PostgreSQL. We want to use `-` as a word separator.)

Reply:
Hai yorokonde!